src/app/components/core/filter/add-filter-wizard/add-filter-wizard.component.html

```diff
@@ -53,9 +55,11 @@
                 <div class="column">
                   <div class="field">
                     <div class="control">
-                      <input class="input" id="name" type="text" placeholder="name" title="No double quotes!" autocomplete="off" formControlName="name" required
-                        (input)="validateServiceName($event)">
-                      <small *ngIf="!isValidName" class="help is-danger level-left">Application name is required</small>
+                      <input class="input" id="name" type="text" placeholder="name" title="No double quotes!"
```

Comment:
why the required is removed?

Reply:
The required is in the .ts file when we are creating form group so removed from here.